core/server/services/members/middleware.js

```diff
@@ -93,9 +93,12 @@ const getMemberSiteData = async function (req, res) {
 };
 
 const createSessionFromMagicLink = async function (req, res, next) {
+    let redirectPath = `${urlUtils.getSubdir()}${req.path}`;
+
     if (!req.url.includes('token=')) {
         return next();
```

Comment:
This should either be removed, or changed to a redirect

Reply:
I guess with https://github.com/TryGhost/Ghost/pull/11795 this is actually okay? :thinking:

Reply:
If we don't have this noone can use /members/ as a route. Your fix changes it so this renders a 404

Reply:
:sleeping: good point